src/js/components/Heading/doc.js

```diff
@@ -72,7 +72,7 @@ export const themeDoc = {
   },
   'heading.level': {
     description:
-      'The level that impacts line height, max width, font size, weight and family of the Heading.',
+      'The level that impacts line height, max width, font size, weight and family of the Heading. Heading level is automatically adjusted at different screen sizes. These screen sizes are derived from the global.breakpoints entry in the theme object.',
```

Comment:
```suggestion
      'The level that impacts line-height, max-width, font size, weight and family of the Heading. Heading level is automatically adjusted at different screen sizes. These screen sizes are derived from its component breakpoints .',
```
I adjusted the text to be more descriptive about the prop rather than getting into the role of the theme here. In my mind, the user could scroll and look for theme breakpoints info in case she is looking for additional information.

Reply:
I think this might be a little deceiving because it makes the phrasing sound like this component controls the breakpoints for screen sizes in general. What if we just simplify it to:
`These screen sizes are derived from global.breakpoints.` The reason being, global.breakpoints exists on the page so if the user did control F or just scrolled they could find elaborated information about it.

Reply:
I think we should avoid the `global` reference on the description hence avoid this declared dependency.
I'm ok with `These screen sizes are derived from breakpoints.`
Or `These screen sizes are derived from the components breakpoints defenitions` or anything along those lines.

Reply:
Okay, that makes sense about avoiding declared dependencies.